### PR TITLE
Cna/divd/divd 2021 00029

### PR DESCRIPTION
**Pull request by MrSeccubus**

### CODE REVIEW

File: 2022/24xxx/CVE-2022-24387.json [hunk 1]
```diff
@@ -1,18 +1,100 @@
 {
```
Comment (cve-team) [inline 2022/24xxx/CVE-2022-24387.json:1]: {} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'product_name': 'SmarterTrack', 'version': {'version_data': [{'version_affected': '<=', 'version_name': '100.0.8019.x', 'version_value': 'Build 8075'}]}}]}, 'vendor_name': 'SmarterTools'}]}}

Comment (cve-team) [inline 2022/24xxx/CVE-2022-24387.json:1]: {} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'CWE-434 Unrestricted Upload of File with Dangerous Type'}]}]}

Comment (cve-team) [inline 2022/24xxx/CVE-2022-24387.json:1]: {} is not allowed for {'reference_data': [{'name': 'https://csirt.divd.nl/DIVD-2021-00029', 'refsource': 'CONFIRM', 'url': 'https://csirt.divd.nl/DIVD-2021-00029'}, {'name': 'https://csrit.divd.nl/CVE-2022-24387', 'refsource': 'CONFIRM', 'url': 'https://csrit.divd.nl/CVE-2022-24387'}]}

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/24xxx/CVE-2022-24387.json (hunk 1 above)
Comment (cve-team) [inline 2022/24xxx/CVE-2022-24387.json:1]: {} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'product_name': 'SmarterTrack', 'version': {'version_data': [{'version_affected': '<=', 'version_name': '100.0.8019.x', 'version_value': 'Build 8075'}]}}]}, 'vendor_name': 'SmarterTools'}]}}

Comment (cve-team) [inline 2022/24xxx/CVE-2022-24387.json:1]: {} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'CWE-434 Unrestricted Upload of File with Dangerous Type'}]}]}

Comment (cve-team) [inline 2022/24xxx/CVE-2022-24387.json:1]: {} is not allowed for {'reference_data': [{'name': 'https://csirt.divd.nl/DIVD-2021-00029', 'refsource': 'CONFIRM', 'url': 'https://csirt.divd.nl/DIVD-2021-00029'}, {'name': 'https://csrit.divd.nl/CVE-2022-24387', 'refsource': 'CONFIRM', 'url': 'https://csrit.divd.nl/CVE-2022-24387'}]}

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by MrSeccubus** [comment]
@cve-team I generated the JSON using Vulnogram and it validates there. Could you give me a pointer as to what is wrong?
The errors only occur on this particular record, not on any others.

HELP!

File: 2022/24xxx/CVE-2022-24387.json (hunk 1 above)
Comment (cve-team) [inline 2022/24xxx/CVE-2022-24387.json:1]: {} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'product_name': 'SmarterTrack', 'version': {'version_data': [{'version_affected': '<=', 'version_name': '100.0.8019.x', 'version_value': 'Build 8075'}]}}]}, 'vendor_name': 'SmarterTools'}]}}

Comment (cve-team) [inline 2022/24xxx/CVE-2022-24387.json:1]: {} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'CWE-434 Unrestricted Upload of File with Dangerous Type'}]}]}

Comment (cve-team) [inline 2022/24xxx/CVE-2022-24387.json:1]: {} is not allowed for {'reference_data': [{'name': 'https://csirt.divd.nl/DIVD-2021-00029', 'refsource': 'CONFIRM', 'url': 'https://csirt.divd.nl/DIVD-2021-00029'}, {'name': 'https://csrit.divd.nl/CVE-2022-24387', 'refsource': 'CONFIRM', 'url': 'https://csrit.divd.nl/CVE-2022-24387'}]}

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by MrSeccubus** [comment]
Found the problem in the end. State was still set to "RESERVED" even tough we meant to publish.

it would be great if the error message was a bit clearer, e.g. :
```
For a "RESERVED" record, {} is not allowed for {'reference_data': [{'name': 'https://csirt.divd.nl/DIVD-2021-00029', 'refsource': 'CONFIRM', 'url': 'https://csirt.divd.nl/DIVD-2021-00029'}, {'name': 'https://csrit.divd.nl/CVE-2022-24387', 'refsource': 'CONFIRM', 'url': 'https://csrit.divd.nl/CVE-2022-24387'}]}
```

Is the validation tool on GitHub too, I'm happy to make a PR for this.

**Pull request merged**